docs/src/cli/delegate-stake.md

```diff
@@ -0,0 +1,136 @@
+# Earn Staking Rewards
+
+After you have [recieved SOL](transfer-tokens.md), you might consider putting
+it to use by delegating stake to a validator node. Solana is a Proof of Stake
+blockchain, which means that holding tokens gives you the option to influence
+which validator nodes participate in securing the ledger. The more tokens you
+hold, the more influence you have. The running assumption of any Proof of Stake
```

Comment:
This isn't just an assumption, it is a fact. The running assumption is that *because* you have more to lose, you are more likely to behave legitimately.

Reply:
the tokens you hold may not be *your* tokens

Reply:
I think this would be better: "The more tokens you hold, the more influence you have, because you have more to lose if a validator node acts maliciously."

Reply:
In this paragraph we talk about "validator nodes" and in the following paragraph, they are "validators".  I think we can drop the word "node" as it doesn't add much here.  As the validator ecosystem continues to evolve, it's possible that single validator entities/people/companies could operate multiple nodes, or some other ambiguity, but validator companies could attract potential stakers to them without needing to describe their "node" or architecture necessarily.